sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -1112,6 +1118,8 @@ def process_bundle(self, instruction_id):
           elif isinstance(element, beam_fn_api_pb2.Elements.Data):
             input_op_by_transform_id[element.transform_id].process_encoded(
                 element.data)
+          # We are done consuming the bundle.
```

Comment:
Actually, "bundle" could be misinterpreted. Maybe call it "set of elements."

Reply:
addressed thanks!